Frontend/research/src/App.js

```diff
@@ -103,35 +209,23 @@ class App extends React.Component {
             />
           )}
         />
+        <div className='bottomFooter'>
+          {localStorage.getItem('token') ? (
+            <Footer/>
+          ) : (<p></p>)}
```

Comment:
you could have null as the second condition instead of an empty p tag if you dont want anything to render if there is no token, you can also use short circuiting and say {localStore.getItem('token') && <Footer /> } if there is no token on local storage it wont run the footer invocation